doc/api/esm.md

```diff
@@ -586,12 +558,27 @@ an all-ES module-syntax version the package. This could be used via `import
 'pkg/module'` by users who are certain that the CommonJS version will not be
 loaded anywhere in the application, such as by dependencies; or if the CommonJS
 version can be loaded but doesn’t affect the ES module version (for example,
-because the package is stateless).
+because the package is stateless). This variant would not require
+`--experimental-conditional-exports`:
+
+<!-- eslint-skip -->
+```js
+// ./node_modules/pkg/package.json
+{
+  "type": "module",
+  "main": "./index.cjs",
+  "exports": {
+    ".": "./index.cjs",
+    "./module": "./wrapper.mjs"
+  }
+}
+```
 
 ##### Approach #2: Isolate State
```

Comment:
Do you think we should also note "./module" as the unflagged approach here too? I like including the conditional exports case first, followed by the progressive approach, but don't want it to get confusing too.